docker/Dockerfile.faux

```diff
@@ -18,15 +18,17 @@ FROM daq/aardvark:latest
 # Run this separately so it can be shared with other builds.
 RUN $AG update && $AG install openjdk-8-jre
 
-RUN $AG update && $AG install isc-dhcp-client ethtool network-manager netcat curl ifupdown
+RUN $AG update && $AG install isc-dhcp-client ethtool network-manager netcat curl python ifupdown
 
 # Prefetch resolvconf to dynamically install at runtime in start_faux.
 RUN $AG update && cd /tmp && ln -s ~/bin bin && $AG download resolvconf && mv resolvconf_*.deb ~
 
 COPY --from=bacnet_build /root/bacnet4j/*.jar bacnet4j/
 
+COPY subset/pentests/brute_server.py pentests/
 COPY misc/start_faux bin/
 COPY misc/bacnet_discover bin/
 
+#CMD cat start_faux_head start_faux_body start_faux_foot > start_faux
```

Comment:
What's this?

Reply:
this has been removed, was part of my tests to extend the Dockerfile.faux